### DISCUSSION TITLE
Mixed Precision

### DISCUSSION THREAD

**Discussion by FrancescoSaverioZuppichini** [onnx/onnx#4713] [Q&A]
Has anyone ever created a mixed precision model?

See https://github.com/microsoft/onnx converter-common/issues/251 and https://github.com/microsoft/onnxconverter-common/issues/252

**Comment by xadupre** [comment]
Issue https://github.com/microsoft/onnxruntime/issues/14235 is also related. It explains why it is long and gives a shortcut.
